source/simplify.d.ts

```diff
@@ -55,4 +76,9 @@ fn(someInterface as Simplify<SomeInterface>); // Good: transform an `interface`
 
 @category Object
 */
-export type Simplify<T> = {[KeyType in keyof T]: T[KeyType]};
+export type Simplify<
+	AnyType,
+	Options extends SimplifyOptions = {},
+> = Flatten<AnyType> extends AnyType
```

Comment:
```suggestion
> = Flatten<AnyType, Options> extends AnyType
```

?

Reply:
No need to pass the options here, it's just for the test, if it doesn't pass without the deep there's no point in going any further